### ISSUE TITLE
Wait state with ResultPath passes validation

### ISSUE THREAD

**Issue by milancermak** [awslabs/statelint#8]
I had a `"Type": "Wait"` state with `"ResultPath": null`. The statelint didn't throw any error, but when I tried to create this state machine in the Step Function console, I got the following error:
```
Error code: SCHEMA_VALIDATION_FAILED
Message: Field ResultPath not allowed. 
```

I think statelint should identify this issue too.

**Comment by timbray** [comment]
I think you're right.

**Comment by timbray** [comment]
Fixed.
